docs/concepts/workers.md

```diff
@@ -4,4 +4,48 @@ title: Workers
 sidebar_label: Workers
 ---
 
-A Worker is a service that hosts the [Workflow](/docs/concepts/workflows) and [Activity](/docs/concepts/activities) implementations.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+A Worker is a service that:
+
+- hosts (registers) [Workflows](/docs/concepts/workflows) and [Activities](/docs/concepts/activities)
+- listens to [Activity Task Queues](/docs/concepts/task-queues) by long polling
+
+Workers should have access to the resources needed to perform the activities they implement, for example:
+
+- network access for external API calls
+- credentials for infrastructure provisioning
+- specialized GPUs for machine learning tasks
+
+Since workers are subscribed to task queues they can implement, it is straightforward to do [Activity Task Routing](/docs/concepts/activities/#activity-task-routing-through-task-queues). 
```

Comment:
I don't think this bit is necessary or helpful. Is task routing something very important that we want to highlight?

Reply:
alright i can take it out, Cully didnt like it either

```suggestion
```